pytest.ini

```diff
@@ -0,0 +1,6 @@
+[pytest]
+#addopts = -m "not migrations"
```

Comment:
Migration tests have a marker; enabling this line will skip those tests on a generic `pytest` call.